tool/site/client/src/components/claim/createDispenseClaim.ts

```diff
@@ -52,10 +52,10 @@ export function createClaim(
   const finalMedicationDispense = medicationDispenses[medicationDispenses.length - 1]
   const prescriptionStatusExtension = getTaskBusinessStatusExtension(finalMedicationDispense.extension)
 
-  const containedPractitionerRole = medicationDispenses[0].contained
-    ?.find(resource => resource?.resourceType === "PractitionerRole") as fhir.PractitionerRole
+  // const tempPractitionerRole = medicationDispenses[0].contained
+  //   ?.find(resource => resource?.resourceType === "PractitionerRole") as fhir.PractitionerRole
```

Comment:
confused if this code is commented? I guess remove it if not in use?

Reply:
Will check if still needed for the Claim work, if not, will remove